job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/PhaseContext.groovy

```diff
@@ -11,14 +11,19 @@ class PhaseContext extends AbstractContext {
 
     String phaseName
     String continuationCondition
+    String executionType
 
     List<PhaseJobContext> jobsInPhase = []
 
-    PhaseContext(JobManagement jobManagement, Item item, String phaseName, String continuationCondition) {
+    PhaseContext(
+        JobManagement jobManagement, Item item, String phaseName, String continuationCondition,
+        String executionType
```

Comment:
Don't add another constructor parameter here. Set the field to the default value.